RedHat-UX/red-hat-design-system
# docs: subnav override of root font size

## What I did

1. Added font-size override for `rh-subnav`

## Testing Instructions

1. View [deploy preview](https://deploy-preview-802--red-hat-design-system.netlify.app/), font-size for subnav should be 16px.

## Notes to Reviewers

This issue stems from the root font size being overridden to something other than the expected `1rem` aka 16px.

See: https://github.com/RedHat-UX/red-hat-design-system/wiki/Root-Font-Size

(**Edit**: wrote the previous commit and notes hastily, the updated version is below after more thoughtfulness)

Currently the issue on `ux.` is that our font-size for content should be 18px, or in token context:  `var(--rh-font-size-body-text-lg, 1.125rem)` given a root font-size set such as `html, body { font-size: 1rem; }`

Given the `html, body {...}` rule from above we can then set`.l-main__content { font-size: var(--rh-font-size-body-text-lg, 1.125rem) }`

This should fix the issue without the need for overrides for any component, given the usage of the `rem` font-sizing from the root.

Look around the [deploy preview site](https://deploy-preview-802--red-hat-design-system.netlify.app/) to make sure this works as intended and no font sizes seem out of place.
